test/Microsoft.NET.Build.Containers.IntegrationTests/EndToEndTests.cs

```diff
@@ -38,7 +38,20 @@ public void Dispose()
         _loggerFactory.Dispose();
     }
 
-    [DockerAvailableFact(Skip = "https://github.com/dotnet/sdk/issues/42850")]
+    // CLI will not let us to target net9.0 anymore but we still need it because images for net10.0 aren't ready yet.
+    // so we let it create net10.0 app, then change the target. Since we're building just small sample applications, it works.
+    public static void changeTargetFrameworkAfterAppCreation(string path)
```

Comment:
does this need to be public?
nit:  ChangeTargetFrameworkAfterAppCreation (with capital C)